addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/IPBridgeHandler.java

```diff
@@ -96,7 +104,24 @@ public IPBridgeConfig getIPBridgeConfig() {
     }
 
     @Override
-    public void handleCommand(ChannelUID channelUID, Command command) {
+    public void handleCommand(final ChannelUID channelUID, Command command) {
+        String scene;
+
+        if (channelUID.toString().contains("pbutton")) {
```

Comment:
I feel silly I did not see this information from your initial commit.  You already went down this path :).  Is the concern that we send a command to a button, but it never gets reset/released by the bridge?  I do not think I am following the reasoning for not allowing buttons to be outputs.  In my instance, I have rules set up that reset the button to its off start after one second after being activated.  I did have a concern that it would cause a loop though (When you activate a button and the bridge echos back that the button is on, would that cause openhab to process/update the channel a second time?

It would be nice to be able to create the channels on the fly based on the info in the discovery process from parsing the XML...but I cannot wrap my head around that at this time.  It works for me if we have all phantom buttons available and I change the label to represent the scene that is calls.